app/Http/Controllers/UserController.php

```
@@ -140,37 +142,44 @@ public function SaveAccount(Request $request): RedirectResponse {
         return redirect()->route('account');
     }
 
+    /**
+     * @api v1
+     * @frontend
+     * @param User $user
+     * @return LengthAwarePaginator|null
+     */
+    public static function statusesForUser(User $user): ?LengthAwarePaginator {
+        if ($user->userInvisibleToMe) {
+            return null;
```

I would recommend to throw an exception instead of returning null. Maybe create something like PermissionException to have it generell and use it on multiple sections?